Sources/Fluent/Model/Model+CRUD.swift

```diff
@@ -95,3 +95,11 @@ extension Future where T: Model {
         }
     }
 }
+
+extension Array where Element: Model {
+    
+    /// See `Model`.
+    public func save(on connectable: DatabaseConnectable) -> Future<[Element]> {
+        return Element.query(on: connectable).create(self)
```

Comment:
Why not just call save multiple times and map it here? I'm guessing you want it to run as a single query?

Reply:
Correct. It's to reduce the number of operations.